src/main/connector/system/Windows/Windows.yaml

```diff
@@ -133,26 +143,68 @@ monitors:
           system.filesystem.usage{system.filesystem.state=used}: $3
   service:
     simple:
-      type: multiInstance
       sources:
-        # Name,ProcessId,StartMode,State
-        source(1):
+        # Name;ProcessId;StartMode;State
+        serviceInformation:
           type: wmi
           namespace: root\CIMv2
-          query: SELECT Name,ProcessId,StartMode,State From Win32_Service
+          query: SELECT Name,ProcessId,State FROM Win32_Service WHERE StartMode != 'Disabled'
+      mapping:
+        source: ${source::monitors.service.simple.sources.serviceInformation}
+        attributes:
+          id: $1
+          processId: $2
+        metrics:
+          system.service.state: $3
+  pageFile:
+    simple:
+      sources:
+        # Name;CurrentUsage;CurrentUsage
+        pageFileUsageInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,CurrentUsage,CurrentUsage FROM Win32_PageFileUsage
+        # Name;MaximumSize
+        pageFileSettingsInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,MaximumSize FROM Win32_PageFileSetting
+        #Name;CurrentUsage;Utilization;Name;MaximumSize
+        pageFileCombinedInformation:
+          type: tableJoin
+          leftTable: ${source::monitors.pageFile.simple.sources.pageFileUsageInformation}
+          rightTable: ${source::monitors.pageFile.simple.sources.pageFileSettingsInformation}
+          leftKeyColumn: 1
+          rightKeyColumn: 1
           computes:
-          - type: translate
+          - type: divide
             column: 3
-            translationTable: ${translation::serviceStatusTranslationTable}
+            value: $5
       mapping:
-        source: ${source::monitors.service.simple.sources.source(1)}
+        source: ${source::monitors.pageFile.simple.sources.pageFileCombinedInformation}
+        attributes:
+          id: $1
+        metrics:
+          system.paging.usage: $2
+          system.paging.utilization: $3

```

Comment:
We're missing some of the recommended semconv metrics about pagefiles ([paging faults](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systempagingfaults) and [paging operations](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systempagingoperations)).

We can use the `Memory` performance object to retrieve these metrics. See Windows' `Pages/s` metric description below:

> Pages/sec is the rate at which pages are read from or written to disk to resolve hard page faults. This counter is a primary indicator of the kinds of faults that cause system-wide delays.  It is the sum of Memory\\Pages Input/sec and Memory\\Pages Output/sec.  It is counted in numbers of pages, so it can be compared to other counts of pages, such as Memory\\Page Faults/sec, without conversion. It includes pages retrieved to satisfy faults in the file system cache (usually requested by applications) non-cached mapped memory files.

Reply:
The problem with these is that they only provide the combined minor/major count, which specifically needs to be separated with system.paging.type. I believe we could get it through perfmon, but that would make this connector use OsCommands instead of WMI.

Reply:
Minor/major is a Linux term (see [this article](https://scoutapm.com/blog/understanding-page-faults-and-memory-swap-in-outs-when-should-you-worry#:~:text=A%20major%20page%20fault%20is,page%20fault%20can%20be%20satisfied.). In Windows, we have soft and hard faults, which corresponds to minor and major respectively, more or less imo.

And we do have separated Soft and Hard faults. And instead of using the minor/major terminology for the `system.paging.type` attribute, I would use `soft` and `hard`.

I know it's not in the semantic convention, but it's reasonable. You could file an issue (or even a PR) with Otel semantic conventions to add soft and hard values for this attribute, for Windows and see what happens.